buildSrc/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ repositories {
 // 'compileJava' task (current target is 17) and 'compileKotlin' task (current target is 1.8) jvm target compatibility should be set to the same Java version.
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
I think the comment above should be fixed as well.

Reply:
Done